### PR TITLE
chore(cli): display spinner before prompting for resource selection

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/override.go [hunk 1]
```diff
@@ -172,16 +173,20 @@ func (o *overrideOpts) askResourcesToOverride() error {
 		return nil
 	}
 
+	o.spinner.Start("Generating CloudFormation template to select resources")
```
Comment (Lou1415926) [inline internal/pkg/cli/override.go]: nit: would users be slightly confused when they saw this? (disclaimer: I am not an english expert 😂 )
```suggestion
	o.spinner.Start("Generating CloudFormation template to select resources from")
```

**Review by Lou1415926 — APPROVED** [review]

File: internal/pkg/cli/task_run_test.go [hunk 2]
```diff
@@ -41,11 +41,24 @@ var defaultOpts = basicOpts{
 	inMemory: 512,
 }
 
-// NOTE: mock spinner so that it doesn't create log output when testing Execute
-type mockSpinner struct{}
+type spinnerTestDouble struct {
```
Comment (huanjani) [inline internal/pkg/cli/task_run_test.go:44]: Sam would be so proud!

File: internal/pkg/cli/override.go [hunk 3]
```diff
@@ -172,16 +173,20 @@ func (o *overrideOpts) askResourcesToOverride() error {
 		return nil
 	}
 
+	o.spinner.Start("Generating CloudFormation template to select resources from")
```
Comment (huanjani) [inline internal/pkg/cli/override.go]: or maybe?
```suggestion
	o.spinner.Start("Generating CloudFormation template for resource selection")
```

Comment (efekarakus) [reply internal/pkg/cli/override.go]: Done!

**Review by KollaAdithya — APPROVED** [review]
lgtm

**Pull request merged**